.github/workflows/industrial_ci_action.yaml

```
@@ -0,0 +1,26 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+# This determines when this workflow is run
+on:
+  # When kinetic-devel branch is pushed to
+  push:
+    branches: [ kinetic-devel ]
+  # When there is a pull request against kinetic-devel
+  pull_request:
+    branches: [ kinetic-devel ]
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: kinetic, ROS_REPO: main}
+          - {ROS_DISTRO: kinetic, ROS_REPO: testing}
```

Please add melodic and noetic as well:
```suggestion
          - {ROS_DISTRO: kinetic, ROS_REPO: main}
          - {ROS_DISTRO: melodic, ROS_REPO: main}
          - {ROS_DISTRO: noetic, ROS_REPO: main}
          - {ROS_DISTRO: noetic, ROS_REPO: testing}
```